modules/likes.php

```diff
@@ -557,24 +557,85 @@ function admin_bar_likes() {
 	}
 }
 
+/**
+ * Callback to get the value for the jetpack_likes_enabled field.
+ */
+function jetpack_post_likes_get_value( array $post ) {
+	$post_likes_switched = get_post_meta( $post['id'], 'switch_like_status', true );
+
+	/** This filter is documented in modules/likes.php */
+	$sitewide_likes_enabled = (bool) apply_filters( 'wpl_is_enabled_sitewide', ! get_option( 'disabled_likes' ) );
+
+	/** This filter is documented in class.json-api-endpoints.php */
+	$is_jetpack = true === apply_filters( 'is_jetpack_site', false, get_current_blog_id() );
```

Comment:
is our use of `get_current_blog_id()` safe here? we ran into issues [in the VideoPress code](https://github.com/Automattic/jetpack/pull/11193/files#diff-db05a9485b8ce5e4945e32ff8cfd34b6R41) because for Jetpack sites it was returning values like `1` and `2` instead of the WordPress.com values returned from `get_option( 'ID' )`

Reply:
Hmm I'm not sure 🤔 I copied it from elsewhere in Jetpack, but now that I look, that turns out to be from within the API. I'll follow up, but am hoping to eliminate the need for it anyway..

Reply:
As far as I can tell we only use the `is_jetpack_site` filter in one place in Jetpack, and we don't rely on the blog ID at all there.
https://github.com/Automattic/jetpack/blob/91d89bc2033bf42fd6993ac712878866f46d4a5b/jetpack.php#L114

We do use it on WordPress.com though, but that should not be an issue there?